packages/framework-core/src/services/read-model-store.ts

```diff
@@ -42,7 +42,7 @@ export class ReadModelStore {
           entitySnapshotEnvelope,
           ' to build new state of read model ${readModelName} with ID ${readModelID}'
         )
-        const newReadModel = this.reducerForProjection(projectionMetadata)(entitySnapshot, readModel)
```

Comment:
Calling the projection function "reducer" can be confused with the entity reducers ;-)